app/controllers/api/v4/blood_sugars_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V4::BloodSugarsController < Api::V3::BloodSugarsController
```

Comment:
I understand this is pulling a lot of existing functionality from its v3 counterpart. However, for consistency, it appears this should inherit from an `Api::V4::SyncController`. How should we handle this?

Reply:
I can add a V4 SyncController that just inherits from the V3 SyncController, we'll end up with some duplication between the V3 and V4 BloodSugarsControllers

Reply:
I think that's fine 👌 If this becomes a problem tomorrow, we can extract the commonalities into modules as appropriate 🤷‍♂